### ISSUE TITLE
bincheck repo refers to TravisCI

### ISSUE THREAD

**Issue by kenliu** [cockroachdb/bincheck#112]
we no longer use this

**Comment by benesch** [comment]
I believe this is still accurate, actually. Travis is necessary to test on macOS and Appveyor on Windows.

**Comment by kenliu** [comment]
Is that something that would work on TeamCity instead of Travis? Trying to rein in the sprawl of dev infrastructure here (which you probably don't know is happening now).

**Comment by benesch** [comment]
It’s definitely possible—I looked into it when I set this repository up,
but it was prohibitively annoying without an official mandate for the
project. You’d need to set up both macOS and Windows TeamCity agents.
Windows TC agents could run on GCE without too much effort, but it’s not
clear where to find macOS cloud agents. At some point I’d set up an old
macOS laptop in a closet as a TC agent, but never had time to see that
through to completion.

Perhaps you have more resources to dedicate to this now, though! It would
sure be nice if it were an automatic part of qualifying releases.
